src/filedef.cpp

```diff
@@ -365,14 +365,17 @@ void FileDef::writeDetailedDescription(OutputList &ol,const QCString &title)
 
 void FileDef::writeBriefDescription(OutputList &ol)
 {
-  if (!briefDescription().isEmpty() && Config_getBool(BRIEF_MEMBER_DESC))
+  if (hasBriefDescription())
   {
     DocRoot *rootNode = validatingParseDoc(briefFile(),briefLine(),this,0,
                        briefDescription(),TRUE,FALSE,0,TRUE,FALSE);
 
     if (rootNode && !rootNode->isEmpty())
     {
       ol.startParagraph();
+      ol.disableAllBut(OutputGenerator::Man);
+      ol.writeString(" - ");
+      ol.enableAll();
```

Comment:
This is not correct. You need to use ol.pushGeneratorState() and ol.popGeneratorState() to avoid that some output format that was already disabled, gets enabled.